### PR TITLE
New package: oksh-0.5.9

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/oksh/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+build_style=gnu-configure
+register_shell="/bin/ksh"
+short_desc="OpenBSD's version of ksh ported to Linux"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="GPL-3"
+homepage="https://connochaetos.org/oksh/"
+distfiles="http://www.connochaetos.org/oksh/oksh-${version}.tar.gz"
+checksum=2323be6676c88bdc916dd206b9e9bd6c23bbada513edaf71dcfad97e8df3c952
+conflicts="ksh>=0"
```
Comment (Vaelatern) [inline srcpkgs/oksh/template]: Please make it alternatives instead?

Comment (diogoleal) [inline srcpkgs/oksh/template]: yep, changed!
```
alternatives="ksh:/usr/bin/ksh"
```

**Comment by chneukirchen** [comment]
That alternatives= is wrong, install oksh as /usr/bin/oksh and then point the alternatives correctly.

Then we need to adjust the ksh and mksh templates.

**Comment by chneukirchen** [comment]
Other ksh adjusted.

File: srcpkgs/oksh/template (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/oksh/template]: Not correct format.

**Comment by diogoleal** [comment]
I fixed alternatives line and name binary e manpage.

File: srcpkgs/oksh/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'oksh'
+pkgname=oksh
+version=0.5.9
+revision=1
+build_style=gnu-configure
+register_shell="/bin/ksh"
```
Comment (chneukirchen) [inline srcpkgs/oksh/template]: I think this should be oksh now too? Tricky...

Comment (ebfe) [inline srcpkgs/oksh/template]: To cover all cases it should add both?

Comment (ebfe) [inline srcpkgs/oksh/template]: ... which won't work either since removing `ksh` will also unregister /bin/ksh.

Comment (Duncaen) [inline srcpkgs/oksh/template]: But it switches to the next available alternative in this case.

**Pull request merged**